app/javascript/components/visualization/controls/ScatterPlotLegend.jsx

```diff
@@ -60,16 +60,38 @@ function LegendEntry({
 
   // clicking the label will either hide the trace, or pop up a color picker
   const entryClickFunction = showColorControls ? () => setShowColorPicker(true) : toggleSelection
+
+  let scatterHoverHandle
+  let scatterHoverId
+  // do not trigger `setActiveTraceLabel()` immediately to prevent triggers on scroll
+  const scatterHoverHandler = (label, currentSelectCheck) => {
+    clearTimeout(scatterHoverHandle)
+    scatterHoverHandle = setTimeout(() => {
+      if (scatterHoverHandle && currentSelectCheck()) {
+        setActiveTraceLabel(label)
+      }
+    }, 10000)
```

Comment:
10 seconds seems too long, on first impression.  To my understanding, this means it'd take 10 s after scrolling for hovered traces to possibly activate (i.e. raise to top layer in scatter plot).  

Is that right?  If so, and if we keep this approach, could we decrease wait to 1 s?

```suggestion
    }, 1000)
```

Reply:
Oh dear, I completely missed this extra `0` and putting the delay down to 1 sec mostly negates the scroll fix (but of course 10 sec makes no sense for a user trying to hover). 

Perhaps an approach of increasing delay as list size increases? I need to take another pass at this as a solution :/